lib/src/core/utils/resources/notification_manager.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter_local_notifications/flutter_local_notifications.dart';
+
+class NotificationManager {
+  void addedFavorite() async {
+    String plainTitle = 'Favorite Added';
+    String plainBody = 'The movie was added to favorites';
+    _showNotification(
+      plainTitle,
+      plainBody,
+    );
+  }
+
+  void removedFavorite() async {
+    String plainTitle = 'Removed Favorite';
+    String plainBody = 'The movie is no longer in favorites';
+    _showNotification(
+      plainTitle,
+      plainBody,
+    );
+  }
```

Comment:
You can create a enum class with the events that you are going to call, and with that change you can avoid to call the same method with different parameters, like this:
`enum class FavoriteEvent {
     add('Favorite Added', 'The movie was added to favorites'),
     remove('Removed Favorite', 'The movie is no longer in favorites');

     const FavoriteEvent(this.title, this.body)
     
     final String title;
     final String body;
}`